plugins/content_distribution/providers/ftp/lib/api/KalturaFtpDistributionJobProviderData.php

```diff
@@ -77,7 +77,14 @@ protected function getDistributionFiles(FtpDistributionProfile $distributionProf
 			if (!$sendMetadataAfterAssets)
 				$files[] = $metadataFile;
 		}
-		
+		$entry =  entryPeer::retrieveByPK($entryDistributionDb->getEntryId());
+		if($entry->getType() == entryType::DATA && !is_null($entry->getDataContent()))
+		{
+			$dataEntryFile = new KalturaFtpDistributionFile();
+			$dataEntryFile->filename = $entry->getName();
```

Comment:
I think it would be better to add a naming method for data entry files, the same way we do with thumbAssets, flavor assets, etc.

Reply:
@hilak Done, please check again